README.md

```diff
@@ -1,11 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
```

Comment:
- [ ] @ginabeki Please just add .env folder to your root directory and then you can use them like this 
![image](https://user-images.githubusercontent.com/74744787/211167785-0071cb70-e563-45e0-a2be-aece894ba87e.png)

and it is working 
![image](https://user-images.githubusercontent.com/74744787/211167799-5a6f0786-2913-4631-9747-2b565db8df07.png)

Reply:
Hi @MeqdamAlqudah , I have implemented the suggested changes, kindly review the updated version